participants/Benxun.md

```diff
@@ -0,0 +1,61 @@
+# Benxun Private Box
+
+
+## (0) Project Name and Team Slack Handles
+
+*Project Name*: Private Box
+
+*Team Slack Handles*: [Ping](https://filecoinproject.slack.com/team/U0153H5EZND)
+
+## (1) Brief Project Description
+
+* A cloud disk app for users to store private data. 
+
+## (2) Link to Project UI
+
+https://demo.benxuntech.cn
+
+## (3) What does your application/UI do?
+
+- Encrypt user's private data and upload to ipfs. 
+- Retrieve data from ipfs and decrypt the data.
```

Comment:
For the final result, you will need to provide  a way for the competition committee to evaluate that the encrypted data.